src/_config-loader-tokens.ts

```
@@ -126,28 +140,59 @@ function loadConfigFile(filePath: string): ConfigData | undefined {
+			logger.info('');
+			logger.info(`Loaded config from: ${configPath}`);
+			logger.info(`  • Schema: ${config.$schema ?? 'none'}`);
+			logger.info(`  • Has defaults: ${config.defaults != null ? 'yes' : 'no'}${config.defaults != null ? ` (${Object.keys(config.defaults).length} options)` : ''}`);
+			logger.info(`  • Has command configs: ${config.commands != null ? 'yes' : 'no'}${config.commands != null ? ` (${Object.keys(config.commands).join(', ')})` : ''}`);
+		}
 		return config;
 	}
 
 	// Auto-discovery from search paths (highest priority first)
+	if (debug) {
+		logger.info('Searching for config files:');
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)